src/components/Content/index.js

```diff
@@ -53,6 +56,10 @@ class Content extends React.Component {
                 <Route path='/my-account' element={<MyAccount />}>
                     <Route path='download' element={<DownloadPage />} />
                 </Route>
+                <Route path='/posts' element={<DetailPost/>} />
+                <Route path='/posts/:id' element={<PostPage/>}>
+                    <Route path='/posts/:id/:comments' element={<CommentPage/>}/>
```

Comment:
если роут вложенный, то можно не дублировать путь
```suggestion
                    <Route path=':comments' element={<CommentPage/>}/>
```